docs/Documentation/Scripting/Building-Blocks/Integration-List.md

```diff
@@ -530,6 +530,31 @@ The syntax is as follows:
 Check the [Luck Perms documentation](https://luckperms.net/wiki/Context)
 for an in-depth explanation on what contexts are and how to add them to permissions.
 
+### Permissions
+If you prefer to directly add or remove permissions without triggering the LuckPerms changelog chat notifications,
+you can utilize the `luckperms addPermission` and `luckperms removePermission` events. 
+
+ ```YAML title="Example"
+ events:
+   addDefaultGroup: "luckperms addPermission permission:group.default,group.quester" #(1)!
+   addNegated: "luckperms addPermission permission:tutorial.done value:false" #(2)!
+   addWithContext: "luckperms addPermission permission:group.legend context:server;lobby" #(3)!
+   addTemporary: "luckperms addPermission permission:donator.level.one expiry:20 unit:MINUTES" #(4)!
+   removeTutorial: "luckperms removePermission permission:tutorial.done"
+   removeMultiple: "luckperms removePermission permission:tutorial.done,group.default" #(5)!
+ ```
+
+1. You can define single or multiple permissions with the `permission` key. You need to separate them with a comma.
+2. You can define Permissions with a optional `value` of `false` to negate them and give them to the player. If you want to override the value of the permission, you can use the `value` argument and set it to `true`.
+3. You can also add optional `context`s to the permissions like `server;lobby`. Read more about contexts [here]
+(https://luckperms.net/wiki/Context). You can define multiple contexts by separating them with a comma.
```

Comment:
You've broken this into a seperate line.
First in my eyes the line break is in a "bad" spot (between the text to click and the link).
Second `1.` and `4.` both don't have the line breakings resulting in an inconsistent source file (even if this does not effect the actual generated file).